.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Maid
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os:
+          - ubuntu-20.04
+          - ubuntu-22.04
+          - macos-11
+          - macos-12
+        ruby-version:
+          - 2.0
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.4
+          - 2.5
+          - 2.6
+          - 2.7
```

Comment:
Ruby 2.7 is the only Ruby listed here that's still supported (and [in two weeks](https://endoflife.date/ruby), only Ruby 3.x will be officially supported).  That said, there's _some_ value in making sure Maid works for older Rubies so that people can easily get started with `maid`.  Is supporting old Ruby versions worth the effort?  I leave that decision to you.

Reply:
I was thinking there might be legacy users still, and since the project was targetting 2.0 I wanted to try and keep that up.

But it largely depends on the GH Actions checks; if it fails for rubies < 2.7 then dropping support for them makes sense anyway since they're EOL (and probably complicated/impossible to install on currently supported OSes)